clang/lib/Frontend/CompileJobCache.cpp

```diff
@@ -538,6 +560,29 @@ bool CompileJobCache::finishComputedResult(CompilerInstance &Clang,
   return true;
 }
 
+Expected<bool> CompileJobCache::mayIngestNonVirtualOutputFromFileSystem(
+    CompilerInstance &Clang) {
+  // FIXME: All consumers should adopt \c llvm::vfs::OutputBackend and this
+  // function should go away.
+
+  const auto &FrontendOpts = Clang.getFrontendOpts();
+  if (FrontendOpts.ProgramAction == frontend::RunAnalysis) {
+    StringRef OutputPath = FrontendOpts.OutputFile;
+    if (OutputPath.empty())
+      return false;
+    if (llvm::sys::fs::is_directory(OutputPath)) {
```

Comment:
And comment and fixme here for what kind of output this is?

Reply:
Done.